### ISSUE TITLE
Feature Request: Ability to set particle color via a "(setq ...)" option

### ISSUE THREAD

**Issue by fxkrait** [elizagamedev/power-mode.el#8]
Particle color is set to match text color:
![image](https://user-images.githubusercontent.com/7407672/233923634-3fab145e-7f1c-4b2c-9a3b-52091a6254d9.png)

I can set it to a custom color (in this case pink) via overriding:
``` elisp
(defun power-mode--foreground-color-before-point ()
  "medium orchid"
)
```
![image](https://user-images.githubusercontent.com/7407672/233926132-504abee3-680d-48d7-8cb4-88137e76d006.png)

- It would be cool if that was available as a configuration option, with a `(setq ...)`. 
   - (I'm not 100% sure how to implement that with the existing code (I guess there could be a if/else added to that function: add a user-settable color variable, if color variable non-nil (user set a color) use that setq'd value & early return/terminate, else implicit else to run existing the code path to determine color from buffer/frame using text color), and with my current level of knowledge/competency (and time), hence why there is no PR appearing from me as of now. But I might push something up in the future regarding this.).

**Comment by elizagamedev** [comment]
Good idea! Should be pretty quick to implement when I've got a moment.

**Comment by fxkrait** [comment]
Added a PR for this, [here](https://github.com/elizagamedev/power-mode.el/pull/9). Just an additional `defcustom` and `if statement`.
- Behavior looks OK, per PR description.
